libs/langchain/langchain/document_loaders/parsers/audio.py

```diff
@@ -136,10 +156,19 @@ def __init__(self, device: str = "0", lang_model: Optional[str] = None):
         # load model for inference
         self.pipe = pipeline(
             "automatic-speech-recognition",
-            model="openai/whisper-medium",
+            model=self.lang_model,  # fix to use model name that was evaluated earlier
```

Comment:
i would remove this comment, it loses meaning outside of this pr

Reply:
Thank you. Fixed.